src/Bundler.js

```diff
@@ -72,6 +72,7 @@ class Bundler extends EventEmitter {
       outDir: Path.resolve(options.outDir || 'dist'),
       outFile: options.outFile || '',
       publicURL: publicURL,
+      isProduction,
```

Comment:
Could u rename this to `production` instead of `isProduction` as we used the key `production` in the vue branch